Cargo.toml

```
@@ -3,6 +3,8 @@ version = "0.2.2"
 authors = ["Gear Technologies"]
 edition = "2021"
 license = "GPL-3.0"
+homepage = "https://github.com/gear-tech/gear"
```

The homepage could be gear-tech.io, I guess.
```suggestion
homepage = "https://www.gear-tech.io"
```